### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by eddiemoore**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,22 +29,18 @@
   },
   "homepage": "https://github.com/migme/bubbly#readme",
   "devDependencies": {
-    "babel": "^5.6.23",
-    "babel-eslint": "^3.1.23",
-    "chai": "^3.2.0",
+    "babel-cli": "^6.6.5",
+    "babel-eslint": "^5.0.0",
```
Comment (bblurock) [inline package.json]: I guess will could bump to `^6.0.0` for better support with ESLint 2?

Comment (eddiemoore) [inline package.json]: Yes we could.

File: package.json [hunk 2]
```diff
@@ -28,22 +29,18 @@
...
-  },
-  "standard": {
-    "parser": "babel-eslint"
-  },
-  "browserify": {
-    "transform": [
-      "babelify"
-    ]
+    "eslint": "^2.4.0",
+    "eslint-config-migme": "^2.0.1",
```
Comment (bblurock) [inline package.json]: In my local I need to add `"eslint-plugin-react": "^5.0.1"` to make eslint run, `eslint-config-migme` seem to use `'eslint-config-migme/rules/react'` by default which need eslint-plugin-react

Comment (eddiemoore) [inline package.json]: Ahh yeah I updated our eslint package with react rules. I'll update this to get it to work without react.

**Comment by bblurock** [comment]
👍

File: package.json (hunk 2 above)
Comment (bblurock) [inline package.json]: Oh I see, It works now with v2.1.0 ~

**Pull request merged**